_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -36,9 +36,15 @@ For examples of how to define scanner rules, see the
 
 ## Bucket ACL scanner
 
-Cloud Storage buckets have ACLs that can grant public access to your
-Cloud Storage bucket and objects. The bucket scanner supports a blacklist mode,
-to ensure unauthorized users don't gain access to your Cloud Storage bucket.
+Access Control Lists (ACLs) are enabled on the Cloud Storage buckets that don't 
+have uniform bucket-level access. ACLs allow you to grant public access to your 
+Cloud Storage bucket and objects. The bucket ACL scanner supports a blacklist 
+mode, to ensure unauthorized users don't gain access to your Cloud Storage bucket.
```

Comment:
It's good to mention the info about uniform bucket-level access. However, the intent was to specify that this scanner ONLY operates on the ACLs of a bucket. It does not scan for IAM policies on the buckets. Please keep the uniform bucket-level access though. The 2nd sentence is a little misleading; I think the previous verbiage about public access is better.

Reply:
Thanks for the comment! This was captured in the `Note` below. However, I rephrased and moved it up!